### PR TITLE
Support polymorphism

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: src/Neo.Compiler.CSharp/Options.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@ namespace Neo.Compiler
     public class Options
     {
         public string? Output { get; set; }
+        public bool Polymorphism { get; set; }
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/Options.cs:6]: Should this be optional?

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/Options.cs:6]: Yes. It is default to `false`.

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/Options.cs]: I've removed the option. Now the compiler will detect whether to enable polymorphism automatically.

**Comment by erikzhang** [comment]
Ready to review.

**Comment by shargon** [comment]
@erikzhang with my last commit, the virtual doesnt' work, could you check it?

https://github.com/neo-project/neo-devpack-dotnet/pull/523/commits/4e6fce5fe0c924345c083a7be0707c83203bfe71

**Comment by erikzhang** [comment]
@shargon Fixed!

**Review by shargon — APPROVED** [review]

**Pull request merged**